grpc/testing/messages.proto

```diff
@@ -319,6 +319,14 @@ message TestOrcaReport {
   map<string, double> utilization = 4;
 }
 
+// Request to a ReturnAll method of the HookService. Specifies the status
+// that will be returned to all pending hook requests as well as hook requests
+// that will arrive after this call.
+message ReturnAllRequest {
```

Comment:
This message does not seem to serve any purpose

Reply:
It is the status that will be returned to hook requests, same as https://github.com/grpc/grpc-proto/blob/master/grpc/testing/messages.proto#L334